x-pack/solutions/security/plugins/security_solution/public/common/components/event_details/get_alert_summary_rows.tsx

```diff
@@ -253,27 +253,36 @@ function getHighlightedFieldsOverride(customs: string[]): EventSummaryField[] {
 /**
  * Assembles a list of fields to display based on the event
  */
-export function getEventFieldsToDisplay({
+export function getHighlightedFieldsToDisplay({
```

Comment:
I know we don't have any right now, but as we're modifying this function, would you be ok writing unit tests for it?
Especially because it's a function used in multiple places...

Reply:
Hey! I added a file that covers custom fields and always displayed fields. I didn't go into each sub functions (events, rule type) because I think then it's just copy and pasting the entire logic. Let me know what you think!